### ISSUE TITLE
[btcd] `runtime: garbage collector found invalid heap pointer`

### ISSUE THREAD

**Issue by adamkrellenstein** [btcsuite/btcd#379]
This an intermittent, fatal error running from `go get github.com/conformal/btcd/...` with `--addrindex`, in Docker, with a base image of `google/golang`. Output to stderr and stdout is here: http://www.pastebin.ca/2974382

**Comment by jrick** [comment]
What version of go?

**Comment by adamkrellenstein** [comment]
```
root@0a8dfcbc7240:/# go version
go version go1.4 linux/amd64
```

**Comment by jrick** [comment]
Duplicate of #222.  Please upgrade to 1.4.2 (this bug was fix in 1.4.1).

**Comment by adamkrellenstein** [comment]
Got it. Thanks!
